src/qt/trafficgraphwidget.cpp

```diff
@@ -1,7 +1,8 @@
-// Copyright (c) 2011-2017 The Vertcoin Core developers
+// Copyright (c) 2011-2018 The Vertcoin Core developers
```

Comment:
I think we should not replace the copyright; only add an additional one to it.

Reply:
Added a Bitcoin copyright to the existing Vertcoin copyright